nessie/src/test/java/org/apache/iceberg/nessie/TestBranchHash.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.nessie;
+
+import com.dremio.nessie.client.NessieClient;
+import com.dremio.nessie.error.NessieConflictException;
+import com.dremio.nessie.error.NessieNotFoundException;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.types.Types;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestBranchHash extends BaseTestIceberg {
+
+  private static final String BRANCH = "test-branch-hash";
+
+  public TestBranchHash() {
+    super(BRANCH);
+  }
+
+  @Test
+  public void testBasicBranch() throws NessieNotFoundException, NessieConflictException {
+    TableIdentifier foobar = TableIdentifier.of("foo", "bar");
```

Comment:
Using the table name "foo.bar" is fine, but using `foobar` as a variable name makes the test harder to read because it isn't obvious what `metadataLocation(catalog, foobar)` does exactly. If foobar is a `Namespace`, then it would be the default location for a table, for example. It would be easier to read if this were something more descriptive, like `tableIdent`.

Reply:
agreed, fixed